### ISSUE TITLE
Susy makes the rebuild process crash

### ISSUE THREAD

**Issue by simonwalsh** [DynamoMTL/dynamo-ops#40]
Example, when reloading a page after a change has been watched : 

```
[18:15:19] Finished 'jekyll-rebuild' after 1.69 ms
[18:15:19] Error: Undefined operation: "1 times valid-gutters(0.25)".
        on line 44 of bower_components/susy/sass/susy/su/_grid.scss, in `span'
        from line 51 of _sass/base/_layout.sass
        from line 26 of ./_sass/app.sass
  Use --trace for backtrace.
[18:15:19] Version: webpack 1.12.13
    Asset    Size  Chunks             Chunk Names
bundle.js  382 kB       0  [emitted]  main
fs.js:849
  return binding.stat(pathModule._makeLong(path));
                                 ^

Error: ENOENT: no such file or directory, stat '/var/folders/q9/r6qgltq10dd_928wxnyk2nf80000gn/T/gulp-ruby-sass/app.css'
    at Error (native)
    at Object.fs.statSync (fs.js:849:18)
    at Immediate.<anonymous> (/Users/simonwalsh/src/dynamo-ops/node_modules/gulp-ruby-sass/index.js:113:12)
    at Immediate.immediate._onImmediate (timers.js:433:18)
    at processImmediate [as _immediateCallback] (timers.js:371:17)
```

**Comment by simonwalsh** [comment]
Need to pull Susy out of the project probably... :(

**Comment by simonwalsh** [comment]
Solved in #41
